h/streamer/messages.py

```diff
@@ -45,13 +44,7 @@ def _handler(payload):
             )
 
     conn = realtime.get_connection(settings)
-    statsd_client = h.stats.get_client(settings)
-    consumer = Consumer(
-        connection=conn,
-        routing_key=routing_key,
-        handler=_handler,
-        statsd_client=statsd_client,
-    )
```

Comment:
`Consumer` here is an [`h.realtime.Consumer`](https://github.com/hypothesis/h/blob/3bdf1540758dc4ac8a359193e37125a57ea3b5ba/h/realtime.py#L13-L75) instance. This PR has removed the `statsd_client` parameter from it